apps/web/components/shared/project-card-variants.tsx

```
@@ -1,384 +1,384 @@
-                    <div key={index} className="p-4 bg-gray-50 rounded-lg">
-                        <div className="flex justify-between items-center mb-1">
-                            <h3 className="font-semibold text-gray-700">{tier.title}</h3>
-                            <span className="bg-purple-100 text-purple-600 px-2 py-1 rounded text-sm">
-                                {tier.left} left
-                            </span>
-                        </div>
-                        <p className="text-gray-500">{tier.support}</p>
-                    </div>
-                ))}
+				{nftTiers.map((tier, index) => (
+					<div key={index} className="p-4 bg-gray-50 rounded-lg">
```

Biome (1.9.4)

[error] Avoid using the index of an array as key property in an element.

This is the source of the key value.

The order of the items may change, and this also affects performances and component state.
Check the React documentation. 

(lint/suspicious/noArrayIndexKey)